helmfile-app/grafana-alloy/values.yaml.gotmpl

```diff
@@ -0,0 +1,155 @@
+          action        = "keep"
+        }
+      }
+
+      prometheus.scrape "custom_targets" {
+        targets = [
+          {
+            "__address__" = "vpn-us1-openvpn-metrics.vpn-us1.svc:9176",
+            "application" = "openvpn",
+            "instance" = constants.hostname,
+          },
+        ]
+        forward_to = [prometheus.remote_write.metrics_service.receiver]
+      }
```

Comment:
The custom targets should be pulled from a helmfile value, since they will diff per cluster/env

Reply:
There's only one cluster here, for now. Noted.